test/cookbooks/osl-git-test/recipes/databag.rb

```diff
@@ -0,0 +1,44 @@
+#
+# Cookbook:: osl-git-test
+# Recipe:: databag
+#
+# Copyright:: 2018, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This recipe tests git credentials from databags
+
+node.default['osl-git']['secrets_databag'] = 'osl-git'
+node.default['osl-git']['secrets_item']    = 'item1'
+
+# Testing default databag (from attributes)
+git_credentials 'root' do
+  notifies :sync, 'git[/root/test]', :immediately
+end
+
+# Testing git repo clone that requires credentials
+git '/root/test' do
+  repository 'https://git.osuosl.org/osuosl/test.git'
+end
+
+# Testing databag specified in properties
+git_credentials 'root' do
+  path '/root/.git-credentials-2'
+  secrets_databag 'osl-git'
+  secrets_item    'item2'
+end
+
+# Testing git repo clone that requires credentials
+git '/root/test2' do
+  repository 'https://git.osuosl.org/osuosl/test.git'
+end
```

Comment:
I'd feel much better if we were testing the same thing between databag and attributes. Can you update this recipe to basically copy what you're doing in the other but obviously adjust for databag?